lib/miq_automation_engine/models/miq_ae_password.rb

```diff
@@ -31,6 +31,10 @@ def self.v1_key
     MiqPassword.v1_key
   end
 
+  def self.all_keys
```

Comment:
Wait... if this inherits from `MiqPassword`, do we need this method defined here?